converter/babeltrace.c

```diff
@@ -70,6 +75,9 @@ static char *opt_input_format, *opt_output_format;
  */
 static GPtrArray *opt_input_paths;
 static char *opt_output_path;
+#ifdef ENABLE_DEBUGINFO
```

Comment:
I would only put the "ifdef" directives where they are absolutely necessary. In this case, that would be the `#include <babeltrace/debuginfo.h>` and the entry in the `long_options` array. The rest will not really have an impact (`free(NULL)`) and should keep the code more readable.